README.md

```diff
@@ -10,7 +10,45 @@ Mobile Verification Toolkit (MVT) is a collection of utilities to simplify and a
 
 [Please check out the documentation](https://mvt-docs.readthedocs.io/en/latest/)..
 
-## Installation
+### Installation using the Docker image
+
+Using Docker is the easiest way of having all the dependencies fixed with a couple of commands.
+Install Docker using the [official instructions](https://docs.docker.com/get-docker/) provided in the Website page.
+
+Afterwards, you can build the Docker image once inside:
+
+```bash
+git clone https://github.com/mvt-project/mvt.git
+cd mvt
+docker build -t mvt .
+```
+
+Once the image is built, can now be tested using, what will prompt a bash terminal:
+
+```bash
+docker run -it mvt
+```
+
+If this is correct, close the container (`exit`) and it is time to connect the Android device to analyse to the USB port using the development mode as explained in the official docs [here](https://developer.android.com/studio/debug/dev-options).
+To have visibility of the USB, the container WILL need to have access to the USB which is not activated in Docker by default.
+This can be done using the `--privileged` parameter when launching Docker as follows and mounting the USB as a volume.
+
+```bash
+docker run -it --privileged -v /dev/bus/usb:/dev/bus/usb mvt
+```
+
+Note that using the `--pivileged` parameter is insecure for a number of reasons explained in detail [here](https://blog.trailofbits.com/2019/07/19/understanding-docker-container-escapes/) as it gives access to the whole system.
```

Comment:
Typo spotted :) `--privileged`

Reply:
Oops. Corrected in the new pushed. Thanks for spotting it! ;)